docker_explorer/lib/storage.py

```diff
@@ -381,6 +385,8 @@ def ShowHistory(self, container_id, show_empty_layers=False):
     # TODO(romaing): Find a container_id from only the first few characters.
     for layer in self.GetOrderedLayers(container_id):
       layer_info = self.GetLayerInfo(layer)
+      if layer is None:
+        raise ValueError('Layer {0} does not exist'.format(layer))
```

Comment:
If you're always expecting`layer` to be a string, please use `{0:s}`.

Reply:
done